src/AlignmentGuides.js

```diff
@@ -684,6 +690,7 @@ class AlignmentGuides extends Component {
 			const id = boxes[box].id || box;
 			const identifier = boxes[box].identifier;  // option index for caption
 			const isSelected = (active === id || activeBoxes.includes(id));
+			id === 'box2' && console.log(active, activeBoxes);
```

Comment:
@ravishchaudhary can we remove the console log ?

Reply:
Sure. Will do.